packages/web-components/fast-element/package.json

```diff
@@ -50,6 +50,7 @@
     "@types/karma": "^5.0.0",
     "@types/mocha": "^7.0.2",
     "@types/webpack-env": "^1.15.2",
+    "@types/web-ie11": "^0.0.0",
```

Comment:
Do we support IE11?

Reply:
Hard to add a reference in the package.json so I anticipated this :). The issue is specifically a break in types that creates an issue with `document.createTreeWalker()`. Updating beyond 4.3.5 creates this issue so this explicitly exists to support the typing. https://github.com/microsoft/TypeScript/issues/33462

This will be removed in `@microsoft/fast-element` v2 as I believe the method is pulled from there.

Reply:
Affirmative. No more use of tree walker in 2.0.